### PR TITLE
[2.13.X] DDF-4722 removed experimental label in docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Removes the "experimental" tag from components that are not experimental.

#### Who is reviewing it? 

@ahoffer @austinsteffes @Kjames5269 

#### Select relevant component teams: 

@codice/docs 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.

@brendan-hofmann
@brjeter
@clockard
@lessarderic
@mcalcote
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested?

verify content
#### What are the relevant tickets?
For GH Issues:
Fixes: #4722 

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/src/main/resources/content/_sources/registry-store.adoc [hunk 1]
```diff
@@ -11,11 +11,6 @@
 
 == {title}
 
-[NOTE]
```
Comment (lessarderic) [inline distribution/docs/src/main/resources/content/_sources/registry-store.adoc]: @mcalcote, does this section and note relate to https://github.com/codice/ddf/blob/2.13.x/catalog/spatial/registry/registry-api/src/main/java/org/codice/ddf/registry/api/internal/RegistryStore.java? If so, should we consider removing the experimental comment in the code as well and move it to a non-internal package?

Comment (mcalcote) [reply distribution/docs/src/main/resources/content/_sources/registry-store.adoc]: I think we should be good to remove it from both places :+1:

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_sources/registry-store.adoc]: done

**Comment by ricklarsen** [comment]
build now

**Pull request merged**